ui/app/components/notice.js

```diff
@@ -99,7 +99,10 @@ Notice.prototype.render = function () {
 
       h('button', {
         disabled,
-        onClick: onConfirm,
+        onClick: () => {
+          this.setState({disclaimerDisabled: true})
```

Comment:
This is only for the button being disabled--it merely re-disables it. Nothing to do with the status of the actual notice being stored in memory.